ksqldb-common/src/main/java/io/confluent/ksql/util/VertxSslOptionsFactory.java

```diff
@@ -126,6 +134,28 @@ public static Optional<PfxOptions> getPfxTrustStoreOptions(final Map<String, Str
     return Optional.empty();
   }
 
+  /**
+   * Returns a {@code KeyStoreOptions} object using the following truststore SSL configurations:
+   * <ul>
+   *  <li>Optional: {@value SslConfigs#SSL_TRUSTSTORE_PASSWORD_CONFIG}</li>
+   * </ul>
+   *
+   * @param props A Map with the truststore location and password configs.
+   * @return The {@code KeyStoreOptions} configured with the above SSL settings.
+   *         Optional.empty() if the truststore password is null or empty.
+   */
+
+  public static Optional<KeyStoreOptions> getBcfksStoreOptions(final Map<String, String> props) {
+
+    final String password = getTrustStorePassword(props);
+
+    if (!Strings.isNullOrEmpty(password)) {
```

Comment:
Don't you need the location obtained like `getPfxTrustStoreOptions`? I see the location is hard-coded in `buildBcvfksOptions`.